Source/WebKit/UIProcess/WebAuthentication/WebAuthenticatorCoordinatorProxy.cpp

```diff
@@ -79,6 +79,10 @@ void WebAuthenticatorCoordinatorProxy::handleRequest(WebAuthenticationRequestDat
         if (result) {
 #if HAVE(UNIFIED_ASC_AUTH_UI)
             if (!authenticatorManager.isMock() && !authenticatorManager.isVirtual()) {
+                if (!ascSuccessfullySoftlinked()) {
+                    handler({ }, (AuthenticatorAttachment)0, ExceptionData { NotSupportedError, "Not implemented."_s });
```

Comment:
Let's not use a c-style cast, let's not cast an int to an enum.  Just have a default value of AuthenticatorAttachment::Platform, even though it's likely meaningless here.